### ISSUE TITLE
CHT - Middleware - CHAT - QA - Get localPK

### ISSUE THREAD

**Issue by josejcb** [Fermat-ORG/fermat#4708]
The problem is when I send a message, to create and identify a chat, we need to send who I am, who the sender is, through, for example, the local PK or PK of the origin device (sender PK). This is because the receiver is waiting the sender PK to identify in his own contact list who is sending the message.

Right now, the agent (chatmiddlewaremonitoragent) try to identify the contact by his PK (sender PK) in the contact table, but we are not sending the correct one because we don't have it (the sender PK), and that's why this agent register a new contact.

In other words, if I have in the receiver contact list the PK = asd123, I must send the message saying that I am the owner of this PK, to identify me in the receiver device.

@darkestpriest  @lostwood @franklinmarcano1970 @GaboHub

**Comment by darkestpriest** [comment]
The local public key is not necessary in the message, we got the chat id from chatmetadata, we can use this Id to fix this issue. If we got the local public key in the chat object and chatmetadata, why we required this in message? I don't get it, we need to be efficient using the design we got in the platform analysis.
